### PR TITLE
Chef

### PR DESCRIPTION
**Pull request by kof**

### CODE REVIEW

File: cssinjs/chef/src/components/Progress.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    alignItems: 'center',
+    justifyContent: 'flex-end'
+  },
+  icon: {
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    zIndex: -1
+  },
+  time: {
```
Comment (sapegin) [inline cssinjs/chef/src/components/Progress.js:26]: Add `font-feature-settings: 'tnum';` to use fixed-width numbers.

File: cssinjs/chef/src/components/StopButton.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    height: '30%',
+    cursor: 'pointer',
+    borderRadius: '10%',
+    background: ({theme}) => theme.colors.primary,
+    willChange: 'opacity',
+    transition:
+      'opacity 0.7s cubic-bezier(0.61, 1, 0.88, 1), background-color 0.2s ease-in-out',
+    '&:hover, &:active': {
+      backgroundColor: ({theme}) => theme.colors.hover,
+      cursor: 'pointer'
```
Comment (sapegin) [inline cssinjs/chef/src/components/StopButton.js]: You already have it in default styles.

File: cssinjs/chef/src/recipes/pourover.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "time": 10,
+      "description": "Pour the rest of the water"
+    },
+    {
+      "time": 5,
+      "description": "Shake the jug"
+    },
+    {
+      "time": 0,
```
Comment (sapegin) [inline cssinjs/chef/src/recipes/pourover.json:29]: You can disable the play button when there's no timer, it feels weird now.

**Review by sapegin — COMMENTED** [review]
I like it! Couple of minor comments ;-)

**Comment by kof** [comment]
Thank you @sapegin !

**Comment by kof** [comment]
Screenshot 2020-04-11 at 13 46 31
this is how the final step looks like now

**Pull request merged**